ome/omero-install
# Centos6 ius

Refactor code to follow the step approach
To test nginx:
- run `./docker-build.sh centos6_py27_ius_nginx/`
- when complete,  `docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_py27_ius_nginx`
- Check that omero and web are running

To test apache22:
- run `./docker-build.sh centos6_py27_ius_apache22/`
- when complete,  `docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_py27_ius_apache22`
- Check that omero and web are running

To test apache24:
- run `./docker-build.sh centos6_py27_ius_apache24/`
- when complete,  `docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_py27_ius_apache24`
- Check that omero and web are running

I have added a "virtual dependencies" step to try to match the current documentation.

still TODO:
- Update readme. To be done in https://github.com/ome/omero-install/pull/81 discussed with @ximenesuk 
- modify `.travis.yml`

The changes above will conflict with existing PRs.
